content/en/docs/concepts/configuration/secret.md

```diff
@@ -371,14 +374,14 @@ spec:
   restartPolicy: Never
 ```
 
-#### Consuming Secret Values from environment variables
+##### Consuming Secret values from environment variables
 
-Inside a container that consumes a secret in an environment variables, the secret keys appear as
-normal environment variables containing the base64 decoded values of the secret data.
+Inside a container that consumes a Secret in an environment variables, the secret keys appear
```

Comment:
Wording:
Inside a container that consumes a Secret through environment variables, the Secret keys ...?

Reply:
Iffy in the original, and I'm happy to save that tweak for a future PR (this is already `size/XL`).